common/helpers/field/index.js

```diff
@@ -241,6 +241,16 @@ function populateAssessmentFields(currentFields, questions) {
   return fields
 }
 
+function processFields(fields, values = {}, errors = {}) {
+  return fromPairs(
+    Object.entries(fields)
+      .map(this.setFieldValue(values))
+      .map(this.setFieldError(errors))
+      .map(this.translateField)
+      .map(this.renderConditionalFields)
```

Comment:
Might want to double check but I updated this recently to include another map.

Reply:
Might also be worth swapping form wizard to use this helper too so we only have that code in one place?

Reply:
Yep. That was the plan. I knew you were updating the other bit and wan't sure whether we'd want to make any of these bits configurable or skippable